onboard/cv/main.cpp

```diff
@@ -88,7 +88,10 @@ int main() {
   arTags[1].distance = -1;
   obstacleMessage.detected = false;
 
-  int tennisBuffer = 0;
```

Comment:
Why are we getting rid of `tennisBuffer`? Wasn't this here to stop noise in thinking we lost the target?